fireatlas/FireObj.py

```diff
@@ -494,6 +499,10 @@ def isignition(self):
     def fireID(self):
         return self._fid
 
+    @fireID.setter
```

Comment:
btw this is the only actual change. All the other stuff was style syntax my IDE made. sorry!

Reply:
yeah that's fine - comes from a lot of this being written in jupyterhub without linting enabled.